File: beginner/java/1079.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {

        Locale.setDefault(Locale.US);

        Scanner scanner = new Scanner(System.in);

        int n, i;
        double media, n1, n2, n3;

        n = scanner.nextInt();

        for(i = 0; i < n; i++){
            n1 = scanner.nextDouble();
            n2 = scanner.nextDouble();
            n3 = scanner.nextDouble();

            media = (n1 * 0.2) + (n2 * 0.3) + (n3 * 0.5);

            System.out.printf("%.1f\n", media);
        }

        scanner.close();

    }
}

File: beginner/c/1079.c
#include <stdio.h>
int main(){

    int n, i;
    double media, n1, n2, n3;

    scanf("%d", &n);

    for(i = 0; i < n; i++){
        scanf("%lf %lf %lf", &n1, &n2, &n3);

        media = (n1 * 0.2) + (n2 * 0.3) + (n3 * 0.5);

        printf("%.1lf\n", media);
    }


    return 0;
    
}